docs/install-nuget-client-tools.md

```diff
@@ -59,7 +62,7 @@ Installation:
 - Visual Studio on Windows: The **NuGet Package Manager** is included with Visual Studio 2012 and later. The Package Manager provides the [Package Manager UI](tools/package-manager-ui.md) and the [Package Manager Console](tools/package-manager-console.md), through which you can run most NuGet operations.
   - The Visual Studio 2017 installer includes the NuGet Package Manager with any workload that employs .NET. To install separately, or to verify that the Package Manager is installed, run the Visual Studio 2017 installer and check the option under **Individual Components > Code tools > NuGet package manager**.
   - The Package Manager UI and Console are unique to Visual Studio on Windows. They are not presently available on Visual Studio for Mac.
-  - Visual Studio does not automatically include the `nuget.exe` CLI, which must be installed separately as described earlier.
+  - Visual Studio does not automatically include a CLI tool, which is required to support NuGet features in the IDE. You can use either the `dotnet` CLI or the the `nuget.exe` CLI, which must be installed separately as described earlier.
```

Comment:
The dotnet SDK is installed automatically in some workloads such as the .NET Core workload. 
Not sure how we want to express that, if at all.